tidy3d/plugins/mode/mode_solver.py

```diff
@@ -926,3 +951,48 @@ def _validate_modes_size(self):
 
     def validate_pre_upload(self, source_required: bool = True):
         self._validate_modes_size()
+
+    @cached_property
+    def reduced_simulation_copy(self):
+        """Strip non-related structures, mediums, monitors, and sources from simulation object.
```

Comment:
instead of "non-related" maybe something about "objects not used by the mode solver"? it wasn't immediately clear what `non-related` meant until I dug into things a little.